extensions/vault/runtime/src/main/java/io/quarkus/vault/internal/client/dto/AbstractVaultAuthAuth.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.vault.internal.client.dto;
+
+import java.util.List;
+
+public class AbstractVaultAuthAuth<METADATA> implements VaultModel {
+
+    public String client_token;
+    public String accessor;
+    public List<String> policies;
+    public List<String> token_policies;
+    public METADATA metadata;
+    public long lease_duration;
+    public boolean renewable;
+    public String entity_id;
+    public String token_type;
```

Comment:
Same for those two above.

Reply:
done https://github.com/quarkusio/quarkus/pull/2897#discussion_r326829173